odd-platform-api/src/main/java/org/opendatadiscovery/oddplatform/service/IngestionServiceImpl.java

```diff
@@ -118,6 +122,35 @@ public Mono<Void> ingest(final DataEntityList dataEntityList) {
             .then();
     }
 
+    @NotNull
+    private Mono<Map<String, DatasetStructureDelta>> getLastStructureDelta(
```

Comment:
What the method is this? Where will you get arguments `latestVersions` and `penaltimateList` ?